docs/MultipleHosts.md

```diff
@@ -68,17 +71,69 @@ host,port,proctype,procname
 ```
 
 ## Environment Variables
-To run this across multiple hosts we need to set the different host names and what environment the host we are on is. To do so I added the following to the setenv.sh file:
+To run this across multiple hosts we need to set the different host names and what environment the host we are on is. Hence the following should be added to the setenv.sh script:
 ```
 export PRIMARYHOSTA=prod.a
 export PRIMARYHOSTB=prod.b
 export SECONDARYHOSTA=dr.a
 export SECONDARYHOSTB=dr.b
 export STACKENV=prod
 export STACKENV_S=dr
-sed -i -e 's/STACKENV/'$STACKENV'/g' ${TORQAPPHOME}/appconfig/process.csv
-sed -i -e 's/STACKENV_S/'$STACKENV_S'/g' ${TORQAPPHOME}/appconfig/nontorqprocess.csv
 ```
 
-Replacing prod.a,prod.b,dr.a and dr.b with the actual hostnames or IPs of the hosts. For the DR hosts I then flipped the values so that for example PRIMARYHOSTA was dr.a and STACKENV was dr.
+Replacing prod.a,prod.b,dr.a and dr.b with the actual hostnames or IPs of the hosts. 
+
+## Environment-Specific Config Script
+
+The below should provide an example of a config script to implement the majority of the above for the production host. It assumes that both a process.csv and nontorqprocess.csv file exist:
+
+```
+# configurable variables
+PRIMARYHOSTA=prod.a
+PRIMARYHOSTB=prod.b
+SECONDARYHOSTA=dr.a
+SECONDARYHOSTB=dr.b
+STACKENV=prod
+STACKENV_S=dr
+
+
```

Comment:
I don't understand everything below here. Why is this all needed? The only thing you should have here are these I think: 

PRIMARYHOSTA=prod.a
PRIMARYHOSTB=prod.b
SECONDARYHOSTA=dr.a
SECONDARYHOSTB=dr.b
STACKENV=prod
STACKENV_S=dr

Can you see if you can make this simpler? All of the below should come as code in the application pack.

Reply:
Had a think about this, I could add a settings file for the rdbq/rdbt processes, versions of the process.csv/nontorqprocess.csv (named something like process_multiplehosts.csv by default), and a script to move the files about appropriately and do the sed stuff to the FSP. Then have the config script set the variables and call the script in the FSP with the variables as the arguments. All of the stuff past this point is the same for both the prod and dr versions so I think that would work.

Reply:
I've made another update here, mainly reordering sections in order for it to proceed as the installtion should do. 

I also updated the config script to do less, it's still copying the environment variables to setenv.sh but I can scrap that as well if you think that's best. The other thing there being that the sed commands for the process.csv/nontorqprocess/csv run in the root install directory as they will be run before the csv files are copied by the user to appconfig.